runhouse/resources/module.py

```diff
@@ -448,7 +457,7 @@ def call_wrapper():
                             **kwargs,
                         )
 
-                    if inspect.isasyncgenfunction(attr) or is_async_gen:
+                    if is_gen:
 
                         async def async_gen():
```

Comment:
I don't quite understand this logic, and why we don't need to create a thread for this, and why we need it for the solo async call

Reply:
Let's talk about this flow live